tests/testthat/test-partial-frame.R

```diff
@@ -54,25 +54,30 @@ test_that("incompatible data frames are an error", {
 })
 
 test_that("dispatch is symmetric with tibbles", {
+  # Now causes infloop with the new double-dispatch mechanism because
```

Comment:
Likely related? https://github.com/r-lib/vctrs/issues/822

Reply:
Probably, this is about calling `vec_ptype2()` again from `vec_ptype2_default()`, for partial objects.